lib/cog/adapters/hipchat.ex

```diff
@@ -20,7 +20,8 @@ defmodule Cog.Adapters.HipChat do
   end
 
   def lookup_user(opts) do
-    HipChat.API.lookup_user(opts)
+    {:ok, user} = HipChat.API.lookup_user(opts)
+    {:ok, Map.put(user, :id, to_string(user.id))}
```

Comment:
This should be coerced when the `ChatProvider` model is created but this will work for now.